prow/recreate_prow_configmaps.py

```diff
@@ -122,8 +117,8 @@ def main():
         "\n!!     WARNING THIS WILL RECREATE **ALL** PROW CONFIGMAPS.     !!"
         "\n!!    ARE YOU SURE YOU WANT TO DO THIS? IF SO, ENTER 'YES'.    !! "
     ) + '\n' + '!' * 65 + '\n\n: '
-    if args.wet:
-        if input(prompt) != "YES":
+    if args.wet and not args.silent:
+        if raw_input(prompt) != "YES":
```

Comment:
Python3 syntax is preferred.

```suggestion
        if input(prompt) != "YES":
```

Reply:
Done.